Gaijinizer patch/Map407.txt

```diff
@@ -4,404 +4,518 @@
 
 結局、全部スフィンクスが仕組んだことだったし
 俺らが関わらなきゃ、こんな大事にはならなかったし
-> CONTEXT: Map407/events/1/pages/0/5/Dialogue/other2 < UNTRANSLATED
+> CONTEXT: Map407/events/1/pages/0/5/Dialogue/other2
+Good grief, what a waste of time this has been.
 
+This whole thing was part of Sphinx's scheme. We
+would've been better off not getting involved.#Ron
 > END STRING
 
 > BEGIN STRING
 ……………そうでもないよ。
 
 ティターニアは、スフィンクスを引き取った見返りに
 人間界への入り口を塞いでくれるって約束したし。
-> CONTEXT: Map407/events/1/pages/0/10/Dialogue/kazuya < UNTRANSLATED
+> CONTEXT: Map407/events/1/pages/0/10/Dialogue/kazuya
+That's not true.
 
+As a reward for taking Sphinx off her hands,
+Titania promised to close that rift.#Ron
 > END STRING
 
 > BEGIN STRING
 少なくとも、この世界で
 あの公園から、妖精に拐われる人間はいなくなる。
 
 果実になった人間は、もうどうしようもないけど…
-> CONTEXT: Map407/events/1/pages/0/15/Dialogue/kazuya < UNTRANSLATED
-
+> CONTEXT: Map407/events/1/pages/0/15/Dialogue/kazuya
+At the very least, no more humans from this world
+will get mixed up with the Fairy Realm. Though
+there's nothing we can do for the ones who've
+already been turned into fruit.#Ron
 > END STRING
 
 > BEGIN STRING
 まぁ………あれはしょうがねぇよ……
 
 結博士に頼めば、あるいは方法があるかも知れないが
 長年やってただけあって、とんでもねぇ数だからな
-> CONTEXT: Map407/events/1/pages/0/20/Dialogue/other2 < UNTRANSLATED
-
+> CONTEXT: Map407/events/1/pages/0/20/Dialogue/other2
+Well, that's just the way it is...
+I don't know whether asking the Professor would
+help, or if there's some other way, but there's
+so many of them it'd take years to get them back.#RonIffy
 > END STRING
```

Comment:
Is this a valid interpretation?
```
Well, that's just the way it is...
Even if we asked the Professor and they found
a way, there's so many it would take years to
turn them all back.
```